crates/scsi/src/main.rs

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: Apache-2.0 or BSD-3-Clause
+
+mod backend;
+mod scsi;
+mod vhu_scsi;
+mod virtio;
+
+fn main() -> backend::Result<()> {
+    backend::scsi_init()
```

Comment:
I don't see much value in creating backend.rs, maybe just merge main.rs and backend.rs.

I had to do it for gpio crate to get rid of some Musl specific configurations.